src/liballoc/collections/binary_heap.rs

```diff
@@ -644,7 +643,7 @@ impl<T: Ord> BinaryHeap<T> {
     /// The remaining elements will be removed on drop in heap order.
     ///
     /// Note:
-    /// * `.drain_sorted()` is O(n lg n); much slower than `.drain()`.
+    /// * `.drain_sorted()` is `O(n * lg(n))`; much slower than `.drain()`.
```

Comment:
I don't know what `lg` is supposed to mean here. Is it `log` with a typo, or is it supposed to indicate a particular base?

Reply:
this is the only place it is used here, i assume it is a typo

Reply:
https://www.math10.com/en/algebra/logarithm-log-ln-lg.html suggests that `lg` == log base 10

https://mathworld.wolfram.com/Lg.html suggests it might mean log base 2

Reply:
I mean anyway the difference between base 2 and base 10 is a constant, so in the context of big-O the base just doesn't matter... but then there are two places in the docs that explicitly give a base in big-O.

Reply:
So, I changed this here to `log` and removed the two explicitly given bases.